### PR TITLE
updating shapecss

### PR DESCRIPTION
**Pull request by MicroDreamIT**
for unminified version

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -7156,7 +7156,11 @@
 			"env": "^Shapecss$",
 			"html": "<link[^>]* href=\"[^\"]*shapecss(?:\\.min)?\\.css",
 			"icon": "Shapecss.svg",
-			"script": "shapecss(?:\\.min)?\\.js",
+			"script": [
+				"shapecss(?:\\-|\\.)([\\d.]*\\d)[^/]*\\.js\\;version:\\1",
```
Comment (gadcam) [inline src/apps.json]: I think replacing <code>(?:\\\\-|\\\\.)</code> with <code>[-.]</code> can make it a bit more readable.

Comment (MicroDreamIT) [inline src/apps.json]: ok. On it sir.

Comment (MicroDreamIT) [inline src/apps.json]: done sir.

**Comment by MicroDreamIT** [comment]
what would be the code if file name look like `shapecss.min.b395c90eed714ca101921.js` ?

**Comment by gadcam** [comment]
To match <code>shapecss.min.b395c90eed714ca101921.js</code> you need to write something like <code>shapecss\\.min\\.[a-f\d]+\\.js</code> or <code>shapecss\\.(?:min\\.)?(?:[a-f\d]+\\.)?js</code>

**Pull request merged**